neuralprophet/plot_model_parameters.py

```diff
@@ -26,7 +26,33 @@
     log.error("Importing matplotlib failed. Plotting will not work.")
 
 
-def plot_parameters(m, forecast_in_focus=None, weekly_start=0, yearly_start=0, figsize=None):
+def check_df_name(m, df_name):
```

Comment:
what is this function used for?
isn't there a function that does the same but for a list - could you use that one for this or merge the two?

Reply:
I believe it is better if we have a different function in this case because we actually have to provide the df_name in this case. We do not extract it from the dict (as the forecaster functions cases) so the errors raised are different. Don't you agree?

Reply:
Sounds reasonable. If this is a sub-case of the other function, the other one could call this one? but if you think they should be separate that's ok, too.